examples/jsm/shaders/OutputShader.js

```diff
@@ -35,7 +31,8 @@ const OutputShader = {
 
 		uniform sampler2D tDiffuse;
 
-		` + ShaderChunk[ 'tonemapping_pars_fragment' ] + ShaderChunk[ 'colorspace_pars_fragment' ] + `
+		#include <tonemapping_pars_fragment>
+		#include <colorspace_pars_fragment>
```

Comment:
Um, I'm not sure why I have written this differently. I could swear previous version required the `ShaderChunk` appraoch otherwise a GLSL error occured.

Reply:
Probably related to automatically included chunks when tonemapping is configured on the renderer.

https://github.com/mrdoob/three.js/blob/bef001a0098c12d5f3c7a7c79b015abff05e9d2b/src/renderers/webgl/WebGLProgram.js#L810

Reply:
Anyway, let's give this a try. It's indeed the cleaner solution.